### ISSUE TITLE
`Layout/LineContinuation*` cops broken on Windows

### ISSUE THREAD

**Issue by ashmaroli** [rubocop/rubocop#11009]
## Expected behavior

Expected the cop(s) to flag offenses.

## Actual behavior

The offenses in below given script did not get reported on my Windows local system but got flagged by GitHub Actions CI running with Ubuntu.

## Steps to reproduce the problem

```ruby
foobar = 'The quick brown fox jumped'\
         ' over the sleeping lazy dog'.
```
Run RuboCop on above script with default config on a ***Windows*** system.

## RuboCop version

```
1.36.0 (using Parser 3.1.2.1, rubocop-ast 1.21.0, running on ruby 2.7.5) [x64-mingw32]
```
